library/bigip_ssl_certificate.py

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/python
+
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# TODO: pkcs12 support when f5-sdk adds it, or convert locally to pem?
+#       Would be nice if f5-sdk file upload could take a string like soap
+#       cert/key import, then we could convert in memory and import
```

Comment:
I'm also a member of the team that devs that, so I will investigate its inclusion. Our release cycle on the sdk is every 2 weeks.